x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/expanded_row.tsx

```diff
@@ -108,28 +147,51 @@ export const ExpandedRow: FC<Props> = ({ item }) => {
   const checkpointing: SectionConfig = {
     title: 'Checkpointing',
     items: checkpointingItems,
-    position: 'left',
+    position: 'right',
   };
 
   const stats: SectionConfig = {
     title: 'Stats',
     items: Object.entries(item.stats.stats).map((s) => {
       return { title: s[0].toString(), description: getItemDescription(s[1]) };
     }),
-    position: 'right',
+    position: 'left',
   };
 
   const tabs = [
+    {
+      id: `transform-settings-tab-${item.id}`,
+      'data-test-subj': 'transformSettingsTab',
+      name: i18n.translate(
+        'xpack.transform.transformList.transformDetails.tabs.transformSettingsLabel',
+        {
+          defaultMessage: 'Settings',
```

Comment:
I'd be tempted to combine the Settings and Details tabs into a single 'Details' tab, with a panel for the 'General' info on the left, and the 'State' and 'Checkpointing' panels to the right. Unless those State / Checkpointing panels can fill up with lots of additional fields depending on the state of the transform? Thoughts @walterra ?

![image](https://user-images.githubusercontent.com/7405507/84896222-79e9b400-b09b-11ea-99dc-18e51110dc7e.png)